Modifications to battle logs summary and telegram notification

virgaux opened the pull request:
Hi Jones, 

I just made some modifications to the battle logs summary. Also fixed the ERC to be more accurate and include the current and new rating after the battle. 

I  added telegram notification if ever the user wants it. This is what it looks like. 

![image](https://user-images.githubusercontent.com/89733547/134594409-f7217e93-39b1-4526-a779-125dd3937644.png)

virgaux committed:
Add telegram notification

*Updated battle summary log
*added telegram notification

Additional installation for telegram notif :

npm install --save telegram-notify

config:
token - your telegram bot token, create bot: https://t.me/BotFather

chatId - telegram chat id for notify, get the id: https://t.me/get_id_bot

virgaux committed:
Minor fix with battle log

* Update much accurate ERC rating
* Fix some logs for better battle summary
* Update New Feature in README
* Add installation guide for Telegram Notification in README

virgaux committed:
Add Telegram Instruction

virgaux committed:
Update README.md

PCJones commented:
I'll modify the telegram part so that nobody has to create a bot, that might take some time.

virgaux commented:
There is a public token that can be used that I made.  This can be used by anyone. It needs more modification to be fancy but it works. The token is 2047824094:AAE0W6PJplxMf9xwPzXJs8CFmQbnp4mBBtY and just look for @Battle_Result_Bot

virgaux committed:
Update README.md

virgaux committed:
Update .env-example

virgaux committed:
Merge branch 'master' of https://github.com/virgaux/ultimate-splinterlands-bot

virgaux committed:
fix some codes

virgaux committed:
Minor fix

virgaux committed:
Add updated DEC after battle

virgaux committed:
Add installation and minor fix

*Add npm install for telegram and chromium installation in intall.bat
* fix some font issue.
* update README

virgaux committed:
small fix

virgaux committed:
Update Install.bat and Index.js

* Update the quest battle log to have accurate info for quest reward.
* Quest reward can be redeemed once available after battle.
* add automation for Install.bat to run stat.bat after installation.

virgaux committed:
Update README.md

virgaux committed:
Update index.js

Fixed the updated Quest status  error

virgaux committed:
Fix Battle errors

- fixed API unable to recognize quest priority
- fixed rented card not being used. (Optional to download)
- Update Install.bat

virgaux committed:
Update README.md

virgaux reviewed (commented):
- API unable to recognize quest priority is still not fixed.

virgaux committed:
another fix

hopefully quest priority is now working cuz I give up. XD

virgaux committed:
fixed quest priority bug

virgaux committed:
Update quests.js

virgaux committed:
Added Features

- Accurate battle summary.
- Can identify if it is draw
- Claim reward function after battle if available.
-added alert notification to telegram

virgaux committed:
Update index.js

virgaux committed:
Update newquests.js

virgaux committed:
update

virgaux committed:
Update index.js

- fix undefined error when selecting cards if api was not used.

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
update api error

virgaux committed:
added feature

virgaux committed:
update api fallback

virgaux committed:
update auto gather

virgaux committed:
Update auto-gather.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update start.bat

virgaux committed:
fix auto gather toggle error

virgaux committed:
fix auto gather toggle

virgaux committed:
Update index.js

virgaux committed:
Fix auto gather high memory

virgaux committed:
Update possibleTeams.js

virgaux committed:
Update .env-example

virgaux committed:
Update README.md

virgaux committed:
update index.js and auto-gather

virgaux committed:
Update auto-gather.js

virgaux committed:
update telegram command

virgaux committed:
Update package.json

virgaux committed:
fix telegram bug

virgaux committed:
Update README.md

virgaux committed:
Update auto-gather.js

virgaux committed:
Merge branch 'master' of https://github.com/virgaux/ultimate-splinterlands-bot

virgaux committed:
team selection bug

virgaux committed:
Update collection power and reward logs

virgaux committed:
update collection and rewards log

virgaux committed:
Update .env-example

virgaux committed:
Update index.js

virgaux committed:
Update newquests.js

virgaux committed:
Update index.js

virgaux committed:
Update README.md

virgaux committed:
Update newquests.js

virgaux committed:
fixed claiming reward bug

virgaux committed:
Update index.js

virgaux committed:
Update auto-gather.js

virgaux committed:
Update package-lock.json

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
update

virgaux committed:
Update install.bat

virgaux committed:
Update possibleTeams.js

virgaux committed:
Update README.md

virgaux committed:
Create history-gather.bat

virgaux committed:
Merge branch 'master' of https://github.com/virgaux/ultimate-splinterlands-bot

virgaux committed:
update

virgaux committed:
fixed tn.sender is not a function bug

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
new log display.

virgaux committed:
Update newquests.js

virgaux committed:
log display

virgaux committed:
Update auto-gather.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
auto switch

virgaux committed:
Update battlesGetData.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update newHistory.json

virgaux committed:
Update newHistory.json

virgaux committed:
Update newHistory.json

virgaux committed:
replace history manual battle data  gather

virgaux committed:
Update auto-gather.js

virgaux committed:
Update auto-gather.js

virgaux committed:
season rewards update

virgaux committed:
Update telnotif.js

virgaux committed:
Update index.js

virgaux committed:
no crash fix

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update auto-gather.js

virgaux committed:
Update auto-gather.js

virgaux committed:
Update auto-gather.js

virgaux committed:
minor update

virgaux committed:
Update auto-gather.js

virgaux committed:
Update newHistory.json

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
update

virgaux committed:
Update auto-gather.js

virgaux committed:
Update telnotif.js

virgaux committed:
Update index.js

virgaux committed:
auto switch update

virgaux committed:
Update start.bat

virgaux committed:
Update newquests.js

virgaux committed:
page error fixed

virgaux committed:
Update index.js

virgaux committed:
priority card added (from yowmama)

virgaux committed:
Update index.js

virgaux committed:
fix bug

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
JS memory limit fixed

virgaux committed:
file size checker for auto gather

virgaux committed:
Fix create new browser from error bug

virgaux committed:
instance issue fix

virgaux committed:
Update index.js

virgaux committed:
manual- autogather with size limit

virgaux committed:
update sh to bat

virgaux committed:
update for faster run

virgaux committed:
update

virgaux committed:
update freezing bug and accurate DEC

virgaux committed:
inaccurate battle result fix

virgaux committed:
Update index.js

virgaux committed:
battledata error fixed

virgaux committed:
Update start.bat

virgaux committed:
Update index.js

virgaux committed:
fixed quest reward bug

virgaux committed:
update memory usage for manual - auto gather

virgaux committed:
Update auto-gather.js

virgaux committed:
Update .env-example

virgaux committed:
test

virgaux committed:
Update index.js

virgaux committed:
reverting back

virgaux committed:
done with testing but revert back to old version

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
season reward fix

virgaux committed:
fixed quest reward bug and added loading visual

virgaux committed:
readline bug fix

virgaux committed:
readline error fix

virgaux committed:
Update index.js

virgaux committed:
Update index.js

virgaux committed:
Update auto-gather.js

virgaux committed:
Update quests.js

virgaux committed:
Update user.js

virgaux committed:
update API

virgaux committed:
fix API call error

virgaux committed:
update API call

virgaux committed:
update API call error fix

virgaux committed:
update

virgaux committed:
ECR accuracy bug fix

virgaux committed:
Create version.txt

virgaux committed:
test

virgaux committed:
Update index.js

virgaux committed:
update

virgaux committed:
Update package.json

virgaux committed:
Update USBpc-Version.txt

virgaux committed:
Update USBpc-Version.txt

virgaux committed:
small update

virgaux committed:
Update index.js

virgaux committed:
Update package-lock.json

virgaux committed:
Create USBpc-Version.json

virgaux committed:
Update USBpc-Version.json

virgaux committed:
update

virgaux committed:
Update USBpc-Version.json

virgaux committed:
update power threshold

virgaux committed:
Update USBpc-Version.json

virgaux committed:
Update index.js

virgaux committed:
update

virgaux committed:
add back up APIs

virgaux committed:
added back up APis

virgaux committed:
Fixed API call response and ECR bug

virgaux committed:
fix bug

virgaux committed:
update

virgaux committed:
fix

virgaux committed:
update

virgaux committed:
Update index.js

virgaux committed:
update

virgaux committed:
update Public API error.

virgaux committed:
update API for card collection

virgaux committed:
Update auto-gather.js

virgaux committed:
update card details

virgaux committed:
Update basicCards.js

virgaux committed:
Update package.json

virgaux committed:
update package

virgaux committed:
Update possibleTeams.js

virgaux committed:
fix can't login error

virgaux committed:
fix can't log in error

virgaux committed:
fix can't login error and make login time instant.

virgaux committed:
fixed unable to login error

virgaux committed:
Update index.js

virgaux committed:
update for the fix for can't login

virgaux committed:
Update splinterlandsPage.js

virgaux committed:
update

virgaux committed:
Update index.js

virgaux committed:
Update USBpc-Version.json

virgaux committed:
minor update

virgaux committed:
Update splinterlandsPage.js

virgaux committed:
minor update

virgaux committed:
update

virgaux committed:
Update index.js

virgaux committed:
Update api.js

virgaux committed:
fixed claim reward error

virgaux committed:
Update cardsDetails.json

virgaux committed:
Update install.bat

virgaux committed:
Update cardsDetails.json

virgaux committed:
Update index.js

virgaux committed:
Update splinterlandsPage.js

virgaux committed:
fixed ongoing login error.

virgaux committed:
Update index.js

virgaux committed:
update for no mana error

virgaux committed:
Delete package-lock.json
